### PR TITLE
Development

### PR DESCRIPTION
**Pull request by SekoViper**
### An Online website for Developer's events. 

- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
- Use GitHub Pages to deploy web pages.
- Apply JavaScript best practices and language style guides in code.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Use objects to store and access data.

link to video recorded
https://www.loom.com/share/b4cae8afb2da4988a5ae79fb76b00a29

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
```
Comment (Kamba56) [inline about.html:1]: - [ ] Please record a video presenting the features of the project. Add a link to the video in your pull request description as demanded by the project requirements.
  - The maximum length of the video is 5 minutes.
  - The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
  - The video contains a demo of the project, showing the pages in different screen sizes.
  - The video contains a highlight about a piece of the code you find interesting or you are proud of making.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    <footer>
+      <div class="footer-container">
+        <div class="logo-container">
+          <img src="img/nextgencodez-logo.png" alt="">
+        </div>
+        <div class="footer-content">
+          <p>2022 creative commons Ghana. All rights Reserved</p>
+        </div>
+      </div>
+    </footer>
```
Comment (Kamba56) [inline about.html]: - [ ] Please make sure the footer of the about page has a dark background to match the design guidelines. You are expected to stick to design guidelines as much as possible.
### Your design


### Design guidelines
f1

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @SekoViper, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Consider making your pull request description a little bit more descriptive. The description should contain a summary of all changes applied to the branch. This will make your work more professional. Kindly check [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more details on professional repo.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by sajid-munawar — APPROVED** [review]
### Hi @SekoViper , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please check my inline comments.
 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**